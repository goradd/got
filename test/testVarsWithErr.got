package testOut

import (
	"bytes"
	"fmt"
)

func givesStringWithErr() (string,error) {
	return "Me with no escaping", nil
}

func givesIntWithErr() (int,error) {
	return -5, nil
}

func givesFloatWithErr() (float32, error) {
	return 6.4, nil
}

func TestVarsWithErr(buf *bytes.Buffer) error {
{{= "Evaluates to a string" }}

{{
Here is the number: {{ie givesIntWithErr() }}
And another: {{se givesStringWithErr() }}
And a float: {{fe  givesFloatWithErr() }}
}}

{{# Here is a comment }}
	return nil // no err
}


