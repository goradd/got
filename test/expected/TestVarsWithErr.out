Evaluates to a string
Here is the number: -5
And another: Me with no escaping
And a float: 6.400000095367432